Sming/Wiring/Print.h

```diff
@@ -26,12 +26,14 @@
 
 #define INITIAL_PRINTF_BUFFSIZE 128
 
+#undef printf_P
```

Comment:
You are right, this is ugly and will cause problems. Therefore I would suggest the following procedure:

* wait for PR #1077 
* optional: implement _printf_P()_ etc. as functions instead a macros in FakePgmSpace (can be changed to an inline wrapper for _printf()_ if #1077 is accepted)
* remove _printf_P()_ from this PR as it is no longer needed
* add printf_P() and println_P() instead